cmd/attack/disk.go

```diff
@@ -70,14 +70,19 @@ func NewDiskWritePayloadCommand(dep fx.Option, options *core.DiskCommand) *cobra
 	}
 
 	cmd.Flags().StringVarP(&options.Size, "size", "s", "",
-		"'size' specifies how many data will fill in the file path with unit MB.")
+		"'size' specifies how many data will fill in the file path with unit.")
 	cmd.Flags().StringVarP(&options.Path, "path", "p", "/dev/null",
 		"'path' specifies the location to fill data in.\n"+
 			"If path not provided, payload will write into /dev/null")
+	cmd.Flags().StringVarP(&options.Unit, "unit", "u", "M",
```

Comment:
can we combine the `size` and `unit` to only one config, just like the `size` in stress attack:

```
-s, --size string       Size specifies N bytes consumed per vm worker, default is the total available memory. One can specify the size as % of total available memory or in units of B, KB/KiB, MB/MiB, GB/GiB, TB/TiB..
```

Reply:
OK，I aggree with you.